packages/core/examples/inputGroupExample.tsx

```diff
@@ -26,33 +27,28 @@ import BaseExample, { handleBooleanChange, handleStringChange } from "./common/b
 export interface IInputGroupExampleState {
     disabled?: boolean;
     filterValue?: string;
+    large?: boolean;
     showPassword?: boolean;
     tagValue?: string;
 }
 
 export class InputGroupExample extends BaseExample<IInputGroupExampleState> {
     public state: IInputGroupExampleState = {
         filterValue: "",
+        large: false,
         showPassword: false,
         tagValue: "",
     };
 
     private handleDisabledChange = handleBooleanChange((disabled) => this.setState({ disabled }));
+    private handleLargeChange = handleBooleanChange((large) => this.setState({ large }));
     private handleFilterChange = handleStringChange((filterValue) => this.setState({ filterValue }));
     private handleTagChange = handleStringChange((tagValue) => this.setState({ tagValue }));
 
     protected renderExample() {
-        const { disabled, filterValue, showPassword, tagValue } = this.state;
+        const { disabled, filterValue, large, showPassword, tagValue } = this.state;
 
-        const clearButton = (
-            <Button
-                className={Classes.MINIMAL}
-                disabled={disabled}
-                iconName="cross"
-                onClick={this.clearFilter}
-                style={{ display: filterValue ? "block" : "none" }}
-            />
-        );
+        const className = large ? Classes.LARGE : "";
```

Comment:
nit: Kind of prefer `classNames({ [Classes.LARGE]: large })` even in this simple case, but don't feel strongly.